docs/azure/sdk/snippets/dependency-injection/HostApplicationBuilder/Program.cs

```diff
@@ -1,15 +1,50 @@
 ﻿using Azure.Identity;
+using Azure.Messaging.ServiceBus;
+using Azure.Messaging.ServiceBus.Administration;
 using Microsoft.Extensions.Azure;
 using Microsoft.Extensions.Hosting;
 
-HostApplicationBuilder builder = Host.CreateApplicationBuilder(args);
+var queueNames = await GetQueueNames();
 
-builder.Services.AddAzureClients(clientBuilder =>
-{
-    clientBuilder.AddSecretClient(new Uri("<key_vault_url>"));
-    clientBuilder.AddBlobServiceClient(new Uri("<storage_url>"));
-    clientBuilder.UseCredential(new DefaultAzureCredential());
-});
+IHost host = Host.CreateDefaultBuilder(args)
+    .ConfigureServices(services =>
+    {
+        services.AddAzureClients(clientBuilder =>
+        {
+            // Register clients for each service
+            clientBuilder.AddSecretClient(new Uri("<key_vault_url>"));
+            clientBuilder.AddBlobServiceClient(new Uri("<storage_url>"));
+            clientBuilder.AddServiceBusClientWithNamespace("<your-namespace>.servicebus.windows.net");
```

Comment:
We use an underscore for the other 2 placeholders. This one uses a dash. Let's stick with one or the other.